pkg/controller/pxc/status.go

```diff
@@ -130,6 +132,13 @@ func (r *ReconcilePerconaXtraDBCluster) updateStatus(cr *api.PerconaXtraDBCluste
 			}
 		}
 		cr.Status.Status = cr.Status.PXC.Status
+	case cr.Spec.ProxySQL == nil && cr.Status.PXC.Status == api.AppStateReady:
```

Comment:
is section can exists but Enabled=false?

Reply:
It will drop into else at 114 line

Reply:
I've added such a case handling